db/schema.rb

```diff
@@ -85,6 +85,8 @@
     t.datetime "created_at",   null: false
     t.datetime "updated_at",   null: false
     t.string   "slug"
+    t.boolean  "status"
```

Comment:
Yeah I also am wondering why there needs to be status and approval boolean columns. A tenant is approved (true or false), but what is status for a tenant? Answer that and I can merge it in.